indico_toolkit/auto_populate/populator.py

```diff
@@ -0,0 +1,294 @@
+import time
+from json import loads
+from typing import List, Tuple, Set
+from indico import IndicoClient
+from indico.types import Workflow
+from indico.queries import CreateExport, DownloadExport, GetDataset, GetModelGroup
+from indico_toolkit.errors import ToolkitPopulationError
+from indico_toolkit.pipelines import FileProcessing
+from indico_toolkit.structure.create_structure import Structure
+
+
+class AutoPopulator:
+    def __init__(self, client: IndicoClient):
+        """
+        Module for manipulating and creating new workflows and teach tasks.
+
+        Args:
+            client (IndicoClient): instantiated Indico Client
+        """
+        self.client = client
+        self.structure = Structure(client)
+        self.file_paths = []
+        self.file_to_class = {}
+        self.file_texts = []
+        self._fp = FileProcessing()
+        self._exceptions = []
+
+    def set_file_paths(
+        self,
+        directory_path: str,
+        accepted_types: Tuple[str] = (
+            "pdf",
+            "tiff",
+            "tif",
+            "doc",
+            "docx",
+            "png",
+            "jpg",
+        ),
+    ) -> None:
+        self._fp.get_file_paths_from_dir(
+            directory_path, accepted_types=accepted_types, recursive_search=True
+        )
+        self.file_paths = self._fp.file_paths
+
+    def create_auto_classification_workflow(
+        self,
+        dataset_name: str,
+        workflow_name: str,
+        teach_task_name: str,
+        labelset_name: str = None,
```

Comment:
What's the utility in naming a labelset? (i.e. most users don't know what a labelset is) guessing we can probably just remove this for simplicity (or give default name?)

Reply:
removed labelset_name